components/AppBar/src/MDCAppBarContainerViewController.m

```diff
@@ -32,6 +32,10 @@ - (instancetype)initWithContentViewController:(UIViewController *)contentViewCon
 
     _contentViewController = contentViewController;
     [self addChildViewController:contentViewController];
+
+    contentViewController.view.frame = self.view.bounds;
+    contentViewController.view.autoresizingMask =
+        UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
```

Comment:
As a general rule of thumb it's dangerous to touch a view controller's `.view` inside of a view controller init method.  This logic should be moved to `viewDidLoad` so that we minimize the window of time that a view is loaded for.